cmd/composition/app/main.go

```diff
@@ -12,10 +12,12 @@ import (
 	"github.com/atlassian/voyager/cmd"
 	"github.com/atlassian/voyager/pkg/composition"
 	"github.com/atlassian/voyager/pkg/util/crash"
+	"k8s.io/klog"
 )
 
 func Main() {
 	rand.Seed(time.Now().UnixNano())
+	klog.InitFlags(nil)
```

Comment:
So we do need to explicitly call `klog.InitFlags` but shouldn't for all API servers (to avoid conflict with API server code), correct?

Reply:
would be nice to raise an issue about this in k/k repo

Reply:
yes to both